config/config.go

```diff
@@ -836,7 +855,7 @@ func (cfg *MempoolConfig) WalEnabled() bool {
 // returns an error if any check fails.
 func (cfg *MempoolConfig) ValidateBasic() error {
 	switch cfg.Type {
-	case MempoolTypeFlood, MempoolTypeNop:
+	case MempoolTypeFlood, MempoolTypeNop, MempoolTypePriority:
```

Comment:
Also from the other comment about supporting the older strings like "v0" and "v1", we will need to make sure they pass validate basic here